### PR TITLE
feat: add nRF Cloud Multi-Service firmware option to 9161 flow

### PR DESCRIPTION
**Pull request by boundlesscalm**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.2.7 - 2024-05-06
+
+### Added
+
+-   option to program and evaluate the nRF Cloud Multi-Service Evaluation
+    firmware on nRF9161 DK and nRF9151 DK.
```
Comment (greg-fer) [inline Changelog.md:6]:
```suggestion
-   Option to program and evaluate the nRF Cloud Multi-Service Evaluation
    firmware on the nRF9161 DK and the nRF9151 DK.
```

File: src/features/device/pca10153.json [hunk 2]
```diff
@@ -80,6 +80,38 @@
                     }
                 ]
             },
+            {
+                "name": "nRF Cloud Multi-Service",
+                "type": "jlink",
+                "description": "Evaluate nRF Cloud Services. Onboarding, data collection, FW updates over the air, logging and alerts.",
```
Comment (greg-fer) [inline src/features/device/pca10153.json:86]: I think we can bet on users knowing what FOTA is.
```suggestion
                "description": "Evaluate nRF Cloud services: onboarding, data collection, FOTA updates, logging, and alerts.",
```

File: src/features/device/pca10153.json [hunk 3]
```diff
@@ -80,6 +80,38 @@
                     }
                 ]
             },
+            {
+                "name": "nRF Cloud Multi-Service",
```
Comment (greg-fer) [inline src/features/device/pca10153.json:84]:
```suggestion
                "name": "nRF Cloud multi-service",
```
Consistency with the sample naming.

File: src/features/device/pca10153.json [hunk 4]
```diff
@@ -179,6 +211,25 @@
                     }
                 ]
             },
+            {
+                "ref": "nRF Cloud Multi-Service",
```
Comment (greg-fer) [inline src/features/device/pca10153.json:215]:
```suggestion
                "ref": "nRF Cloud multi-service",
```

File: src/features/device/pca10171.json [hunk 5]
```diff
@@ -80,6 +80,38 @@
                     }
                 ]
             },
+            {
+                "name": "nRF Cloud Multi-Service",
+                "type": "jlink",
+                "description": "Evaluate nRF Cloud Services. Onboarding, data collection, FW updates over the air, logging and alerts.",
```
Comment (greg-fer) [inline src/features/device/pca10171.json:86]: As in comments above.

File: src/features/device/pca10171.json [hunk 6]
```diff
@@ -179,6 +211,25 @@
                     }
                 ]
             },
+            {
+                "ref": "nRF Cloud Multi-Service",
```
Comment (greg-fer) [inline src/features/device/pca10171.json:215]: As in comments above.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: Changelog.md (hunk 1 above)
Comment (boundlesscalm) [reply Changelog.md:6]: Should I not also replace `nRF Cloud Multi-Service` with `nRF Cloud multi-service` here?

Comment (greg-fer) [reply Changelog.md]: Yes:
```suggestion
-   Option to program and evaluate the nRF Cloud multi-service 
    firmware on the nRF9161 DK and the nRF9151 DK.
```

File: src/features/device/pca10153.json [hunk 7]
```diff
@@ -80,6 +80,38 @@
                     }
                 ]
             },
+            {
+                "name": "nRF Cloud multi-service",
+                "type": "jlink",
+                "description": "Evaluate nRF Cloud services: onboarding, data collection, FOTA updates, logging and alerts.",
```
Comment (greg-fer) [inline src/features/device/pca10153.json:86]:
```suggestion
                "description": "Evaluate nRF Cloud services: onboarding, data collection, FOTA updates, logging, and alerts.",
```

File: src/features/device/pca10171.json [hunk 8]
```diff
@@ -80,6 +80,38 @@
                     }
                 ]
             },
+            {
+                "name": "nRF Cloud multi-service",
+                "type": "jlink",
+                "description": "Evaluate nRF Cloud services: onboarding, data collection, FOTA updates, logging and alerts.",
```
Comment (greg-fer) [inline src/features/device/pca10171.json:86]:
```suggestion
                "description": "Evaluate nRF Cloud services: onboarding, data collection, FOTA updates, logging, and alerts.",
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**